firedrake/matrix.py

```diff
@@ -121,21 +114,9 @@ def assemble(self):
                                   has been removed.  Use 'assemble(a, bcs=bcs)', which\
                                   now returns an assembled matrix.")
 
-    @property
-    def M(self):
-        """The :class:`pyop2.Mat` representing the assembled form
-
-        .. note ::
-
-            If you just need a handle on the :class:`pyop2.Mat` object it's
-            wrapping, use :attr:`_M` instead."""
-        # User wants to see it, so force the evaluation.
-        self._M._force_evaluation()
-        return self._M
-
     def force_evaluation(self):
         "Ensures that the matrix is fully assembled."
-        self._M._force_evaluation()
+        pass
```

Comment:
Probably retained for compatibility, although most calls to this elsewhere are being removed.

Reply:
Removed these calls.